lib/bolt/executor.rb

```diff
@@ -18,14 +18,18 @@ class Executor
     attr_reader :noop, :transports
     attr_accessor :run_as
 
+    # FIXME: There must be a better way
+    # https://makandracards.com/makandra/36011-ruby-do-not-mix-optional-and-keyword-arguments
     def initialize(concurrency = 1,
                    analytics = Bolt::Analytics::NoopClient.new,
                    noop = nil,
-                   bundled_content: nil)
+                   bundled_content: nil,
+                   load_config: true)
       @analytics = analytics
       @bundled_content = bundled_content
       @logger = Logging.logger[self]
       @plan_logging = false
+      @load_config = load_config || true
```

Comment:
Will this ever be false? `false || true` will be `true`. I think this can just be `@load_config = load_config`.